apis/fluentd/v1alpha1/plugins/params/const.go

```diff
@@ -51,6 +51,7 @@ const (
 	ForwardInputType InputType = "forward"
 	TailInputType    InputType = "tail"
 	SampleInputType  InputType = "sample"
+	MonitorAgentType InputType = "monitorAgent"
```

Comment:
```
<source>
  @type monitor_agent
  port 24220
  bind 0.0.0.0
  tag example
  emit_interval 5
  include_config true
  include_retry true
</source>
```
so this could be `MonitorAgentType InputType = "monitor_agent"`?